test/e2e/nft/erc721-interaction.spec.js

```diff
@@ -125,6 +125,12 @@ describe('ERC721 NFTs testdapp interaction', function () {
           windowHandles,
         );
 
+        // avoid race condition
```

Comment:
Can you precise what race condition may occour here?

Reply:
The test may try to find the 6th nft on the extension before it is loaded.

Reply:
Oh, ok. It should probably be in the comment, or not have the comment. I was a bit confused by it.